packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -24,7 +24,7 @@ import {
 
 interface Breadcrumb {
   facetId: string;
-  label: string;
+  label?: string;
```

Comment:
```suggestion
  label: string;
```

I don't think that's needed anymore?

Reply:
what should we assign label in facetBreadcrumbs() if we can't assign undefined to it?

Reply:
you can give it an empty string